src/arc.ts

```diff
@@ -627,6 +629,18 @@ export class Arc extends GraphNodeObserver {
     Logger.debug(`Data saved successfully as ${descriptionHash}`)
     return descriptionHash
   }
+
+  public approveTokens(tokenAddress: Address, spender: Address, amount: BN) {
+    const erc20Abi = this.getABI({ abiName: 'ERC20Mock', version: LATEST_ARC_VERSION })
```

Comment:
I prefer not support/use ERC20Mock.
you can create the tokenContract just with the `approve` abi.(have it hard coded).